common-api/src/main/java/org/pmiops/workbench/cdr/dao/ConceptDao.java

```diff
@@ -14,11 +14,24 @@
 
 
     @Query(value = "select c.* from concept c " +
-            "where c.domain_id in ('Condition','Observation','Procedure', 'Measurement', 'Drug') and " +
-            "c.standard_concept=:standard_concept " +
+            "where c.domain_id in ('Condition','Observation','Procedure', 'Measurement', 'Drug') " +
             "order by c.count_value desc limit :maxResults ",
     nativeQuery = true)
-    List<Concept> findAllConceptsOrderedByCount(@Param("standard_concept") String standard_concept, @Param("maxResults") long maxResults);
+    List<Concept> findAllConceptsOrderedByCount(@Param("maxResults") long maxResults);
+
+    @Query(value = "select c.* from concept c " +
+            "where c.domain_id in ('Condition','Observation','Procedure', 'Measurement', 'Drug') and " +
+            "c.standard_concept='S' " +
+            "order by c.count_value desc limit :maxResults ",
+            nativeQuery = true)
+    List<Concept> findAllStandardConceptsOrderedByCount(@Param("maxResults") long maxResults);
+
+    @Query(value = "select c.* from concept c " +
```

Comment:
Shouldn't these say "where count_value > 0 || source_count_value > 0 " ? 

Have you tested with the public-ui ?